cli/commands/info/index.ts

```diff
@@ -0,0 +1,118 @@
+): CommandHandler {
+    assertExists(args, 'args')
+    assertExists(ethersAgentRegistryProvider, 'ethersAgentRegistryProvider')
+    assertExists(agentRegistry, 'agentRegistry')
+    assertExists(agentRegistryContractAddress, 'agentRegistryContractAddress')
+    assertExists(getFromIpfs, 'getFromIpfs')
+    assertExists(getLogsFromPolyscan, 'getLogsFromPolyscan')
+
+    return async function info() {
+        const finalAgentId = args.agentId ? args.agentId : agentId;
+
+        assertIsNonEmptyString(finalAgentId, 'agentId');
+
+        console.log(`Fetching bot info...`);
+
+        const [agent, currentState] = await Promise.all([ 
+             await agentRegistry.getAgent(finalAgentId), 
+             await agentRegistry.isEnabled(finalAgentId) as boolean,
+        ]);
+
+        
+        const ipfsMetaHash = agent.metadata;
+
+        const ipfsData = await getFromIpfs(ipfsMetaHash)
+        printIpfsMetaData(ipfsData,currentState)
+
+        console.log(`Recent Activity: \n`);
+
+
+        const eventTopicFilters = AGENT_REGISTRY_EVENT_FRAGMENTS
+            .filter(fragment => (isRelevantSmartContractEvent(fragment.name)))
+            .map(eventFragment => {
+                return {
+                    type: eventFragment.name,
+                    topicHash: getTopicHashFromEventName(eventFragment.name as StateChangeContractEvent)
+                } as EventFilter
+            });
+
+
+
+        const logs: PolyscanLog[] = [];
+
+        await Promise.all(eventTopicFilters.map(async filter => {
+            const eventLogs = await getLogsFromPolyscan(agentRegistryContractAddress, filter, finalAgentId);
+            logs.push(...eventLogs)
+        }))
+
+        const filteredLogs = filterSimultaneousEventsOnBotCreation(logs)
+
+        filteredLogs.sort((logOne, logTwo) => logOne.timeStamp < logTwo.timeStamp ? 1 : -1)
+
+        
+        for (let log of filteredLogs) {
+            const eventName = getEventNameFromTopicHash(log.topics[0]);
+            console.log(` ${formatDate(new Date(log.timeStamp * 1000))} ${formatEventName(eventName)} by ${ipfsData.from} (https://polygonscan.com/tx/${log.transactionHash})\n \n`)
+        }
+    }
+}
+
+const printIpfsMetaData = (ipfsData: IpfsMetadata, botStatus: boolean) => {
+    const formattedData = formatIpfsData(ipfsData, botStatus)
+    console.log("\n")
+    Object.entries(formattedData).forEach(([key, value]) => console.log(`${key}: ${value}`))
+    console.log("\n")
+}
+
+const formatEventName = (eventName: string): string => {
+    if(eventName === "Transfer") {
+        return "Bot Created";
+    }
+
+    return eventName.replace("Agent", "Bot ");
+}
+
+const formatDate = (date: Date): string => {
+    const monthFormatter = new Intl.DateTimeFormat('default', {month: 'short'})
+    const dayFormatter = new Intl.DateTimeFormat('default', {day: '2-digit' })
+    return `[${dayFormatter.format(date)} ${monthFormatter.format(date)} ${date.toTimeString()}]`
```

Comment:
can we format the date like so: `10 May 2022 07:33:40 GMT`. keeping everything GMT and without the day of the week